tests/sentry/dynamic_sampling/test_prioritise_projects.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime, timezone
+
+from freezegun import freeze_time
+
+from sentry.dynamic_sampling.prioritise_projects import fetch_projects_with_total_volumes
+from sentry.snuba.metrics import TransactionMRI
+from sentry.testutils import BaseMetricsLayerTestCase, SnubaTestCase, TestCase
+
+MOCK_DATETIME = datetime(2023, 8, 7, 0, 0, 0, tzinfo=timezone.utc)
```

Comment:
Considering that we use Snuba, I suppose we have the 90 days TTL, thus hardcoding time will result in a failure 3 months from now due to data being deleted right after insertion.

I would suggest using the following snippet:
```python
MOCK_DATETIME = (timezone.now() - timedelta(days=1)).replace(
    hour=0, minute=0, second=0, microsecond=0
)
```

Reply:
fixed.